academic-services/knowledge-exploration-service/tutorial-schema-design.md

```diff
@@ -278,44 +280,48 @@ For more detailed deployment instructions, See [Create API Instances](get-starte
 
 ## Create a client application that uses the MAKES API instance
 
-Now that we have deployed a MAKES API instance that uses the custom index build, the last step is to create a frontend client to that enables users to browse and filter library publications. In the remaining sections of the tutorial, we will be using the sample UI code to illustrate how to retrieve data and generate filters via MAKES Evaluate and Histogram APIs.
+Now that we have deployed a MAKES API instance that uses the custom index, the last step is to create a frontend client that gives user the ability to browse and filter library publications. In the remaining sections of the tutorial, we will be using the sample UI code to illustrate how to retrieve data and generate filters via MAKES Evaluate and Histogram APIs.
 
 ### Explore the library browser application sample
 
 The code for the library browser application sample is part of the standard MAKES deployment. After the MAKES API instance with the custom index build has been fully deployed, you can use the library browser application by visiting the following URL:
 
-`<Makes_Instance_Url>/examples/privateLibraryExample/privateLibraryExample.html`
+`<Makes_Instance_Url>/examples/privateLibraryExample/index.html`
 
```

Comment:
Any chance you can include a .zip file with all the files in the distribution and just have them download that?

It's physically painful to have to craft the URL for each file below and download them separately.